pyproject.toml

```diff
@@ -1,3 +1,80 @@
 [build-system]
-requires = ["setuptools>=40.8.0", "wheel"]
+requires = ["setuptools>=40.8.0", "setuptools-scm", "wheel"]
 build-backend = "setuptools.build_meta"
+
+
```

Comment:
Maybe this should be a single blank line? Alternatively suppose we could use two blank lines between all sections

```suggestion

```